### ISSUE TITLE
lib needed to work on an M0

### ISSUE THREAD

**Issue by gallaugher** [BlitzCityDIY/partyParrotAdventures#1]
Thanks for what looks like such a fun example + which should be far easier to implement that my feeble old-man fuddles. I have a 32x32 linked to an Adafruit LED Matrix Shield, which is linked to a Metro M0. I must be missing something very basic in the "lib" file for CircuitPython 5 on the m0, because when I try running the code I get:
  File "code.py", line 5, in <module>
ImportError: no module named 'rgbmatrix'
Doing a quick search in the "lib" folder for CP 5.0 extracted from CircuitPython.org, I don't see a file named rgbmatrix.
And when I try to run simple Hello Word-style code at:
https://learn.adafruit.com/circuitpython-display-support-using-displayio/text
I get:
  File "code.py", line 6, in <module>
AttributeError: 'module' object has no attribute 'DISPLAY'
Could you kindly provide info on "lib" configuration or anything other special config I may be missing to use this with an Metro M0 and an Adafruit RGB Matrix Shield for Arduino.
Again - sorry to trouble you + thanks!

**Comment by BlitzCityDIY** [comment]
hello! rgbmatrix is part of the core, which means that it's included in the uf2 bootloader by default like time, board, etc., so I would try updating the version of CP that you're running on the m0 to 5.3 just in case since it's fairly new: https://circuitpython.org/board/metro_m0_express/

**Comment by gallaugher** [comment]
Yes, this is the version I've got. As fresh as fresh can be. Perhaps it's not clear, but maybe the M0s don't support this product for CircuitPython? I may not be doing this right, but I tried this from the REPL:
import board
dir(board)
and get:
['__class__', 'A0', 'A1', 'A2', 'A3', 'A4', 'A5', 'D0', 'D1', 'D10', 'D11', 'D12', 'D13', 'D2', 'D3', 'D4', 'D5', 'D6', 'D7', 'D8', 'D9', 'I2C', 'MISO', 'MOSI', 'NEOPIXEL', 'RX', 'SCK', 'SCL', 'SDA', 'SPI', 'TX', 'UART']
no DISPLAY, which should be there, right? This is why the basic Hello World code isn't working and might be related to why I can't get the M0 with shield working with what I'm sure is awesome code you've written, above? What HW did you use? Thanks!

**Comment by BlitzCityDIY** [comment]
gotcha, okay tried with my M0 (had prototyped with a feather m4, metro M4 and M4 airlift) and had same issue. and you hypothesized correctly, looks like the M0 does not support the rgbmatrix in its core unfortunately: https://circuitpython.readthedocs.io/en/latest/shared-bindings/support_matrix.html

I'll make a note of this in the documentation for this. Not sure if you have access to an M4 but that should work plug and play.
